services/exam-manager/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM scanhub-base
+FROM ghcr.io/brain-link/scanhub/scanhub-base:latest
```

Comment:
Can we make this conditional somehow? For development one would  like to use the local image. Otherwise a developer always needs to push changes to the main/ghcr branch to see the effects of the updated base image?

Reply:
This applies to any use of the base image